contrib/tracking.cpp

```diff
@@ -0,0 +1,87 @@
+#include "tracking.h"
+#include <opencv2/opencv.hpp>
+
+
+bool Tracker_Init(Tracker self, Mat image, Rect boundingBox) {
+    cv::Rect2d c_boundingBox(boundingBox.x, boundingBox.y, boundingBox.width, boundingBox.height);
+
+    bool ret = (*self)->init(*image, c_boundingBox);
+    return ret;
+}
+
+
+bool Tracker_Update(Tracker self, Mat image, Rect *boundingBox) {
+    cv::Rect2d cBox;
+    bool ret = (*self)->update(*image, cBox);
+    boundingBox->x = int(cBox.x);
+    boundingBox->y = int(cBox.y);
+    boundingBox->width = int(cBox.width);
+    boundingBox->height = int(cBox.height);
+    return ret;
+}
+
+
+
+
+TrackerMil TrackerMil_Create() {
+    return new cv::Ptr<cv::TrackerMIL>(cv::TrackerMIL::create());
+}
+
+void TrackerMil_Close(TrackerMil self){
+    delete self;
+}
+
+
+TrackerBoosting TrackerBoosting_Create() {
+    return new cv::Ptr<cv::TrackerBoosting>(cv::TrackerBoosting::create());
+}
+
+void TrackerBoosting_Close(TrackerBoosting self){
+	delete self;
+}
+
+
+TrackerMedianFlow TrackerMedianFlow_Create() {
+    return new cv::Ptr<cv::TrackerMedianFlow>(cv::TrackerMedianFlow::create());
+}
+
+void TrackerMedianFlow_Close(TrackerMedianFlow self){
+	delete self;
+}
+
+
+TrackerTld TrackerTld_Create() {
```

Comment:
Could this be `TrackerTLD` since the `cv` namespace or class is `TrackerTLD`?

Reply:
camel or not ?

(they were all like it is in opencv, origially)

Reply:
but yes, happy to change it back !

Reply:
I dont know, I like the Go way (`TrackerTLD`, etc.) but it is up to you. What matters the most is to be consistent.